docs/docs/FAQ/Machine-Learning-FAQ.md

```diff
@@ -0,0 +1,125 @@
+:::
+:::tip
+You can check how much model processing power will be required by checking Flops(B) in [this table](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_results.csv)
+The higher the value this indicates increased consumption of server resources.
+:::
+
+* For Facial recognition the best model will be `buffalo_l`.
+
+<details>
+  <summary>why is it?</summary>
+
+from this [deepinsight/insightface#1820](https://github.com/deepinsight/insightface/issues/1820) it looks like the "r100 glint360k model" (which seems [antelopev2](https://github.com/deepinsight/insightface/blob/master/python-package/README.md)) performs worse than the `buffalo_l` model because the latter is trained on more data with a better training schedule. Seems like there is a reason that insightface does not promote the `antelopev2` model and prefers the `buffalo_l`[[1](https://github.com/deepinsight/insightface/issues/1820#issuecomment-968200625)].
+
+Glint360k (of `antelopev2`) is 360k identities and 17M images.
+WebFace600K(600K identities) is same as WebFace12M(12 million images), a subset of WebFace42M(cleaned WebFace260M)[[2](https://github.com/deepinsight/insightface/issues/1770#issuecomment-927131443)].
+
+See the [full discussion](https://github.com/immich-app/immich/discussions/5028) in GitHub for more information.
+
+</details>
+
+* For the smart search feature the best Clip model will be `ViT-H-14-378-quickgelu` or `vit-H-14-Qickgelu`.
+
+<details>
+  <summary>why is it?</summary>
+
+You might want to consider `ViT-H-14-quickgelu` over `ViT-H-14-378-quickgeludfn5b`
+because it takes much less processing power and the difference between the
+Average perf. on 38 datasets is really "nothing"
+to be precise a difference of 0.0118.
+And the advantage of ViT-H-14-quickgelu is that it requires much less processing power
+381.68 compared to 1054.05.
+
+Sources:
+[Discord](https://discord.com/channels/979116623879368755/1174254430221254738),
+[openclip results](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_results.csv#L3),
+[openclip retrieval results](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_retrieval_results.csv).
+
+</details>
+
+<details>
+  <summary>How to apply?</summary>
+
+1. In the browser, log in to your Admin user
+2. Go to the url [http://localhost:2283/admin/system-settings?open=job-settings](http://localhost:2283/admin/system-settings?open=job-settings)
+3. Go to Machine Learning Settings
+4. change the models and save
+5. rerun the jobs of ML to take effect
+
+</details>
+
+:::note
+Changing a model does not delete the previous model, pay attention to delete the old model to save space in drive.
+Models are stored in IMMICH_MODEL-CACHE.
+:::
+
+### I would like to use the Clip model i created, how can i do that?
```

Comment:
I've seen a few recurring questions about it on Discord at different times, I assumed that maybe it could go into the FAQ (maybe another new "advanced" page?)

Reply:
In that case, change the question to "Can I use a custom model?" with the answer "No, this is not supported. Only models listed in the [Hugging Face org](https://huggingface.co/immich-app) are compatible. Feel free to make a feature request if there's a model not listed here that you think should be added."